scripts/e2e/Ingredients.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Recipe Page Tests', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+    await page.fill('input[name="username"]', 'bob_lee');
+    await page.fill('input[name="password"]', 'password');
+
+    // Click login button
+    await page.locator('button[type="submit"]').click();
+
+    // Wait for login response
+    await page.waitForResponse((response) => response.url().includes('/authenticate/login'));
+  });
+
+  test('Recipe creation', async ({ page }) => {
+    // Click the first Inventory button
+    await page.locator('a:has-text("Inventory")').nth(0).click();
+
+    // Click New ingredient button
+    await page.locator('button:has-text("New ingredient")').click();
+
+    // expect url to be inventory/new
+    expect(page.url()).toContain('/inventory/new');
+
+    await page.locator('input[placeholder="e.g. banana"]').fill('pepper');
+    await page.locator('input[placeholder="e.g. 35"]').fill('3');
+    await page.locator('#root__layout > div.layout__has__sidebar.MuiBox-root.css-7d7xpv > main > div > form > div.MuiGrid-root.MuiGrid-container.MuiGrid-spacing-xs-3.css-zow5z4-MuiGrid-root > div:nth-child(3) > div > div > input').fill('g');
```

Comment:
it is bad practice to locate a component like this as it is very brittle. you should find the component via its property (or if there isn't one, you can add a `data-testid` param to the component to locate it)